client/src/app/page/list/accordion/item/item.component.ts

```diff
@@ -13,49 +13,72 @@ import { User } from '../../../../common/models/user';
 export class ItemComponent implements OnInit {
   @Input() task: Task;
   user = new User();
+  userId: string;
   users: User[];
   checkedResolve: boolean;
   cssClass: string;
   cssClassVisible: string;
   markResolve: boolean;
   alertMessage: string;
-  isOpen: boolean;
   unreadClass: string;
-  unreadOpenClass: string;
   taskIsOpen: boolean;
+  userType: string;
 
   constructor(private readonly router: Router,
               private readonly route: ActivatedRoute,
               private readonly userService: UserService,
               private readonly tasksService: TasksService) { }
 
   ngOnInit(): void {
-    this.loadUser();
-    this.isOpen = this.task.isOpen;
+    this.userService.getUser()
+      .subscribe(user => {
+        this.user = user;
+        this.changeClassUnread();
+    });
     this.userService.getAll()
       .subscribe(users => this.users = users);
     this.cssClass = '';
     this.taskIsOpen = false;
-    this.unreadClass = 'unread';
+    this.unreadClass = '';
+    this.userType = this.userService.getUserType();
+    this.userId = this.userService.getUserId();
   }
 
-  openTask(): any {
+  openTask(): void {
     this.taskIsOpen = true;
     this.changeClassUnread();
     this.checkedAuthorOrPerformer();
   }
 
   changeClassUnread(): void {
-    this.taskIsOpen ? this.unreadClass = 'unread-open' : this.unreadClass = 'unread';
+    if (this.userType === 'hr') {
```

Comment:
try to divide it to more private methods
![image](https://user-images.githubusercontent.com/48734701/57436363-31c95280-7248-11e9-847d-eb5eaa6397c4.png)

Reply:
example:
isHr(...)
isDeveloper(...)